prep-javascript-variables/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>JavaScript Variables</title>
+  </head>
+  <body>
+    <script>
+      const fullName = 'Bryant Mendoza';
+      console.log(fullName);
+      console.log(typeof fullName);
+      const numberOfPets = 1;
+      console.log(numberOfPets);
+      console.log(typeof numberOfPets);
+      const likesVegetables = true;
+      console.log(likesVegetables);
+      console.log(typeof likesVegetables);
+      let empty;
+      console.log(empty);
+      console.log(typeof empty);
+      const unset = null;
```

Comment:
These should be swapped. `empty` should be getting assigned `null`, while `unset` shouldn't get assigned anything

Reply:
gotcha ! i just changed it.